metricflow/sql_clients/sql_utils.py

```diff
@@ -65,12 +65,17 @@ def make_df(  # type: ignore [misc]
     )
 
 
-def make_sql_client(url: str, password: str) -> AsyncSqlClient:
-    """Build SQL client based on env configs. Used only in tests."""
+def dialect_from_url(url: str) -> SqlDialect:
```

Comment:
Was not expecting this here, it'd be good to put the need for this change in the commit message.

Reply:
Updated.